wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -868,13 +870,25 @@ data Account = Account
     , accWalletId  :: !WalletId
     } deriving (Show, Ord, Eq, Generic)
 
+-- | Datatype wrapping addresses for per-field endpoint
+newtype AccountAddresses = AccountAddresses
+    { acaAddresses :: [WalletAddress]
+    } deriving (Show, Ord, Eq, Generic)
+
+-- | Datatype wrapping balance for per-field endpoint
+newtype AccountBalance = AccountBalance
+    { acbAmount    :: V1 Core.Coin
```

Comment:
Bikeshedding -- `acbAmount` or `acbBalance`? 🚴 

I suspect the latter reads better in the output JSON -- what do you think?

Reply:
Weak vote for `acbBalance` but I think both are fine

Reply:
:upside_down_face: ... I went for `acbBalance` at first, because it reads better. Then I notice the fireld was actually named `amount` in the `Account` representation. Therefore, I changed for `acbAmount` as well for consistency.